rollup.config.js

```diff
@@ -6,30 +6,35 @@ import uglify from 'rollup-plugin-uglify';
 import babel from 'rollup-plugin-babel';
 import replace from 'rollup-plugin-replace';
 import strip from 'rollup-plugin-strip';
-import yargs from 'yargs';
 
-const args = yargs.argv;
+const min = process.env.NODE_ENV === 'min';
 
 export default {
-  input: './lib/index.js',
+  input: './src/index.js',
   output: {
-    file: `dist/react-beautiful-dnd${args.min ? '.min' : ''}.js`,
+    file: `dist/react-beautiful-dnd${min ? '.min' : ''}.js`,
     format: 'umd',
     name: 'ReactBeautifulDnd',
     globals: { react: 'React' },
   },
   plugins: [
-    babel({ exclude: 'node_modules/**', babelrc: false }),
-    resolve(),
-    commonjs(),
+    babel({
+      exclude: 'node_modules/**',
+      runtimeHelpers: true,
```

Comment:
why runtimeHelpers and not external ones? where external is ofc a misleading name here, because they would be included in the bundle

Reply:
It was. But guys want to keep polyfills.

Reply:
For future safety: rather than adding polyfills we can delegate to babel

Reply:
But if you think that is a terrible idea I am open to being convinced!